official/utils/arg_parsers/parsers.py

```diff
@@ -201,6 +202,13 @@ def __init__(self, add_help=False, num_parallel_calls=True, inter_op=True,
           metavar="<MTS>"
       )
 
+    if benchmark_log_dir:
+      self.add_argument(
+          "--benchmark_log_dir", "-bld", default=None,
+          help="[default: %(default)s] The location of the benchmark logging.",
+          metavar="<BLD>"
+      )
+
```

Comment:
I think this makes more sense not as part of Performance; we will want all (or nearly all) models to be benchmarkable, but only a subset will be actively perf optimized. Not opposed to having as separate, or to including with the hooks above; that will force us to make sure all models do this correctly more quickly.